marshmallow/compat.py

```diff
@@ -1,10 +1,31 @@
 # -*- coding: utf-8 -*-
 import sys
 import itertools
+import functools
+import inspect
 
 PY2 = int(sys.version_info[0]) == 2
 PY26 = PY2 and int(sys.version_info[1]) < 7
 
+
+def get_func_args2(func):
```

Comment:
okay, sounds fine to me. And thanks for helping me out with how to do the rebase.